_posts/2018-09-05-cockpit-177.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 177
+author: pitti
+date: 2018-09-05
+tags: cockpit linux
+slug: cockpit-177
+category: release
+summary: Cockpit with LUKSv2 support and FreeIPA ssh known hosts integration
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 177.
+
+### Storage: Support LUKS v2
+
+Cockpit's Storage page can now create, lock, unlock, resize, and otherwise
+configure encrypted devices using the
+[LUKS v2](https://www.saout.de/pipermail/dm-crypt/2017-December/005771.html)
+format.  This offers more flexible unlocking policies, stronger cryptography,
+and better compatibility with future changes and features.
```

Comment:
> Cockpit's Storage page can now create, lock, unlock, resize, and otherwise
configure encrypted devices using the [LUKS v2](https://www.saout.de/pipermail/dm-crypt/2017-December/005771.html) format.
>
> This new version of LUKS offers more flexible unlocking policies, stronger cryptography, and better compatibility with future changes. Additionally, support for LUKS v2 in Cockpit allows for Network Bound Disk Encryption (NBDE) through Clevis and Tang.

Reply:
This is not completely correct - clevis&tang and NBDE work wiht LUKSv1 as well, just (until now) *not* with LUKSv2. I "merged" your and my proposals accordingly.